web/src/main/java/org/springframework/security/web/server/csrf/CookieServerCsrfTokenRepository.java

```diff
@@ -164,6 +167,32 @@ public void setSecure(boolean secure) {
 		this.secure = secure;
 	}
 
+	/**
+	 * Sets maximum age in seconds for the cookie that the expected CSRF token is saved to
+	 * and read from. By default maximum age value is -1.
+	 *
+	 * <p>
+	 * A positive value indicates that the cookie will expire after that many seconds have
+	 * passed. Note that the value is the <i>maximum</i> age when the cookie will expire,
+	 * not the cookie's current age.
+	 *
+	 * <p>
+	 * A negative value means that the cookie is not stored persistently and will be
+	 * deleted when the Web browser exits.
+	 *
+	 * <p>
+	 * A zero value causes the cookie to be deleted immediately therefore it is not a
+	 * valid value and in that case an {@link IllegalArgumentException} will be thrown.
+	 * @param cookieMaxAge an integer specifying the maximum age of the cookie in seconds;
+	 * if negative, means the cookie is not stored; if zero, the method throws an
+	 * {@link IllegalArgumentException}
+	 * @since 5.7
```

Comment:
Will you please change this to 5.8? It cannot be added in 5.7 since future maintenance releases for 5.7 are only for bug fixes.

Reply:
Sure gimme a sec

Reply:
@jzheaux I already added the fix for the @ since, but I tried doing a `git rebase 5.8.x` but it's having quite a lot of conflicts since I checkout the branch from main, is there any other way to avoid all that? I don't want to mess up something else.

Reply:
Instead, you could checkout `5.8.x`, do a cherry-pick for your commit on `gh-11432`, delete the `gh-11432` and recreate it from `5.8.x`:

```
git checkout 5.8.x
git cherry-pick `git log --pretty=format:"%h" -n1`
git log -n1 5.8.x # make sure the commit is on the branch before proceeding
git branch -D gh-11432
git checkout -b gh-11432
```

Reply:
@jzheaux Woah, that's quite nice, I've always been the kind that just use `git rebase` and solve conflicts all the way lol.

Thank you, I don't know if it needs anything else